distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -346,6 +346,36 @@ grant codeBase "file:/platform-parser-xml" {
     permission java.lang.RuntimePermission "setContextClassLoader";
 }
 
+grant codeBase "file:/catalog-core-localstorageprovider" {
+    permission java.io.FilePermission "${ddf.home.perm}content", "write";
+    permission java.io.FilePermission "${ddf.home.perm}content${/}-", "read, write";
+}
+
+grant codeBase "file:/platform-country-converter-local" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}fipsToIso.properties", "read";
+}
+
+grant codeBase "file:/org.ops4j.pax.web.pax-web-extender-war" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}application-definitions", "read";
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}application-definitions${/}*", "read";
+}
+
+grant codeBase "file:/catalog-core-solrcommands/security-sts-attributequeryclaimshandler/catalog-confluence-source/org.eclipse.jetty.websocket.common" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}*", "read";
+}
+
+grant codeBase "file:/security-sts-attributequeryclaimshandler/security-sts-ldapclaimshandler" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}attributeMap.properties", "read";
+}
+
+grant codeBase "file:/security-servlet-session-expiry/org.eclipse.jetty.websocket.common" {
+    permission java.io.FilePermission "${ddf.home.perm}etc${/}ws-security${/}server${/}*", "read";
+}
+
+grant codeBase "file:/platform-solr-server-standalone" {
```

Comment:
❗️ This bundle is already referenced once in the policy file and the full set of permissions it needs (including this new one) are on the row for `grant codeBase "file:/catalog-core-standardframework"`.

Should remove this row, then move `platform-solr-server-standalone` from its current row to the standard framework row.

Reply:
Done